LabeledItem/LabeledItem.js

```diff
@@ -9,140 +9,23 @@
  * @exports LabeledItemBase
  */
 
-import compose from 'ramda/src/compose';
-import kind from '@enact/core/kind';
-import {MarqueeBase as Marquee, MarqueeController} from '@enact/ui/Marquee';
-import PropTypes from 'prop-types';
-import Pure from '@enact/ui/internal/Pure';
 import React from 'react';
-import Spottable from '@enact/spotlight/Spottable';
-import Touchable from '@enact/ui/Touchable';
 
 import Icon from '../Icon';
-import {ItemBase} from '../Item';
-import Skinnable from '../Skinnable';
-
-const Controller = MarqueeController(
-	{marqueeOnFocus: true},
-	Skinnable(
-		ItemBase
-	)
+import Item from '../Item';
+
+// eslint-disable-next-line enact/prop-types
+const LabeledItemBase = ({children, titleIcon, ...rest}) => (
```

Comment:
I forgot we talked about adding warnings so I'll push an update to these shims to address that.

Reply:
Added `deprecate` to each of these.